css/media.css

```diff
@@ -0,0 +1,1201 @@
+@media only screen and (max-width: 1024px) {
+	.header-content ,
+	.poster-content ,
+	.product-content {
+		max-width: 1024px;
+	}
+
+	.header-container ,
+	.poster-container ,
+	.product-container ,
+	.product-items {
+		max-width: 930px;
+	}
+
+	.poster-container{
+		height: 252px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project_Axion/pullRequest?prid=5457134)